src/client/datascience/jupyter/kernels/kernelDependencyService.ts

```diff
@@ -131,10 +138,13 @@ export class KernelDependencyService implements IKernelDependencyService {
         });
         const installPrompt = isModulePresent ? Common.reInstall() : Common.install();
         const selectKernel = DataScience.selectKernel();
+        // Due to a bug in our code, if we don't have a resource, don't display the option to change kernels.
+        // https://github.com/microsoft/vscode-jupyter/issues/6135
+        const options = resource ? [installPrompt] : [installPrompt, selectKernel];
```

Comment:
I'm removing the ability to switch kernels when using `Jupyter` (non-RAW), fixing this requires a big refactor or we can just hack/patch the code, either way, I'm not keen on doing that at this stage.
TLDR - we're not passing the `resource`, how/why can be dealt with #6135